src/autocomplete/autocomplete.scss

```diff
@@ -21,7 +21,7 @@
 // Wrapper for the input and result elements to ensure alignment
 .autocomplete-body {
   position: relative;
-  display: inline-block;
+  display: inline;
```

Comment:
This was originally changed to `inline-block` to remediate this bug we saw in Firefox:
<img width="287" alt="Screenshot of listbox overlapping with input field in firefox" src="https://user-images.githubusercontent.com/16447748/160480258-7f4c7c83-d091-497e-b316-74a9e5111417.png">


[Preview of bug](https://primer-css-9dvu53q33-primer.vercel.app/css/components/autocomplete)

However, `inline-block` results in the listbox overflowing out of its container. See https://github.com/primer/view_components/pull/1050#issuecomment-1079309276. We found the overlap issue does not appear when testing the swapped instances in prod in FireFox. Though this bug surfaces in Primer CSS docs, we don't see it when listbox is hidden THEN appears, so I think we should move forward with this to remediate the overflow issue.